microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -117,79 +117,98 @@ For more information, navigate to [Understanding browser versions and WebView2](
 
 Only one installation of the Evergreen WebView2 Runtime is needed for all Evergreen apps on the device.  Several tools are available at [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2#download-section) to help you deploy the Evergreen Runtime.
 
-*   For online clients: _WebView2 Runtime Bootstrapper_ is a tiny (approximately 2 MB) installer.  The WebView2 Runtime Bootstrapper downloads and installs the Evergreen Runtime from Microsoft servers that matches the user's device architecture.
-    *   In the setup part of your WebView2 app, link to the bootstrapper.  Use a link to programmatically download the bootstrapper; select the **Get the Link** button at the above download page.
-    *   Or, download the bootstrapper and package it with your WebView2 app.
+* For online clients: _WebView2 Runtime Bootstrapper_ is a tiny (approximately 2 MB) installer.  The WebView2 Runtime Bootstrapper downloads and installs the Evergreen Runtime from Microsoft servers that matches the user's device architecture.
+  * In the setup part of your WebView2 app, link to the bootstrapper.  Use a link to programmatically download the bootstrapper; select the **Get the Link** button at the above download page.
+  * Or, download the bootstrapper and package it with your WebView2 app.
+
+* For offline clients: _WebView2 Runtime Standalone Installer_ is a full installer that installs the Evergreen WebView2 Runtime in offline environments.
 
-*   For offline clients: _WebView2 Runtime Standalone Installer_ is a full installer that installs the Evergreen WebView2 Runtime in offline environments.
+### Installing the Runtime as per-machine or per-user
 
-Currently, both the bootstrapper and standalone installer only support per-machine installs, which requires elevation of permissions.  If an installer is run without elevation, the user is prompted to elevate permissions.
+<!-- keep the 3 instances or variants of this passage in sync: -->
+
+The latest bootstrapper and standalone installer support both _per-machine_ and _per-user_ installs of the WebView2 Runtime.
+
+If you run the installer from an elevated process or command prompt, the Runtime is installed as _per-machine_.  If you don't run the installer from an elevated process or command prompt, the Runtime will be installed as _per-user_.  However, a _per-user_ install is automatically replaced by a _per-machine_ install, if a _per-machine_ Microsoft Edge Updater is in place.  A _per-machine_ Microsoft Edge Updater is included as part of Microsoft Edge, except for the Canary preview channel of Microsoft Edge.
```

Comment:
@liminzhu - Is Edge Updater always inherently per-machine, or are there two versions of it, a per-user version & a per-machine version?

Reply:
It depends on how the browser or WV is getting installed. If Edge is installed per machine (and for Stable, Beta, Dev, it is per-machine) then it's per machine. Canary is installed per user, so the updater is per user as well. What I mean to say in this paragraph is really if you have per-machine installed Edge (which should be the cases for all channels except Canary), the per machine updater that comes with it will put a per-machine WebVIew2 Runtime to replace the per-user one at some point.

Reply:
@liminzhu - please review my added clarification here of the 1st instance of the paragraph.  In instance 2 & 3 of the paragraph, I added a link pointing to this section, to read the added detail.

Reply:
@mikehoffms thanks! I love having a `### Installing the Runtime as per-machine or per-user` that is referenced by other parts of the doc.

```
For the Stable channel of Microsoft Edge, the Beta preview channel of Microsoft Edge, and the Dev preview channel of Microsoft Edge, Microsoft Edge is installed per-machine, and so Microsoft Edge Updater is installed per-machine. If you have installed Microsoft Edge per-machine, the per-machine Microsoft Edge Updater that comes with Microsoft Edge installs a per-machine WebView2 Runtime to replace the per-user WebView2 Runtime at some point.

The Canary preview channel of Microsoft Edge is installed per-user, and so Microsoft Edge Updater is installed per-user.
```

This part does feel a bit repetitive as we're re-iterating what we were saying in the two sentences above that (`However, a _per-user_ install is automatically replaced by a _per-machine_ install, if a _per-machine_ Microsoft Edge Updater is in place.  A _per-machine_ Microsoft Edge Updater is included as part of Microsoft Edge, except for the Canary preview channel of Microsoft Edge.`). Replacing per-user with per-machine install is also not an important aspect that a developer must dwell on, and more of an optimization on our end, so I'd suggest keeping the two sentences and getting rid of the long version so that readers move on to other content faster.

Reply:
I have one more edit about regkey I want to squeeze in.

Reply:
@mikehoffms see the new commit i've made and let me know what you think about the above and whether we're ready to merge. Thanks!

Reply:
will do

Reply:
"keeping the two sentences and getting rid of the long version"
Removed the long version.